apps/storybook/src/stories/cyberstormComponents/CodeInput.stories.tsx

```
@@ -0,0 +1,58 @@
+    value: "",
+    placeholder: "Code here...",
+    validationBarProps: { status: "waiting" },
+  },
+} satisfies Meta<typeof CodeInput>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = { args: {} };
+export const Value: Story = { args: { value: 'console.log("example log");' } };
+export const Error: Story = {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)